js/src/jupyter-leaflet.js

```diff
@@ -344,6 +345,40 @@ var LeafletVideoOverlayView = LeafletRasterLayerView.extend({
     },
 });
 
+var LeafletHeatmapView = LeafletLayerView.extend({
+
+    create_obj: function () {
+        this.obj = L.heatLayer(
+            this.model.get('latlngs'),
+            {
+                minOpacity: this.model.get('minOpacity'),
+                maxZoom: this.model.get('maxZoom'),
+                maxIntensity: this.model.get('maxIntensity'),
+                radius: this.model.get('radius'),
+                blur: this.model.get('blur'),
+                gradient: this.model.get('gradient')
+            },
+            this.get_options()
+        );
+    },
+
+    model_events: function () {
+        LeafletHeatmapView.__super__.model_events.apply(this, arguments);
+
+        this.listenTo(this.model, 'change:options', function () {
+            latlngs = this.model.get('latlngs');
```

Comment:
Unused here, you can remove it. You need a special event handler for the `latlngs`, because it's not an option.